zkevm-circuits/src/evm_circuit/execution/create.rs

```diff
@@ -576,6 +630,11 @@ impl<F: Field, const IS_CREATE2: bool, const S: ExecutionState> ExecutionGadget<
     ) -> Result<(), Error> {
         let opcode = step.opcode.unwrap();
         let is_create2 = opcode == OpcodeId::CREATE2;
+        println!(
+            "offset {} rw_counter {}, is_create2 {}",
+            offset, step.rw_counter, is_create2
+        );
```

Comment:
we need clean the println and many useless commented codes before merging

Reply:
yes, will clean up today!

Reply:
cleaned .